src/database/sql_database.h

```diff
@@ -172,6 +173,9 @@ class SQLDatabase : public Database {
     /// \brief migrate resources from mt_cds_objects to grb_resource before removing the column (DBVERSION 13)
     bool doResourceMigration();
     void migrateResources(int objectId, const std::string& resourcesStr);
+    
+    /// \brief returns a fmt-prinable identifier name
+    SQLIdentifier identifier(const std::string_view& name) const { return SQLIdentifier(name, table_quote_begin); }
```

Comment:
You should add `table_quote_end` also, e.g. M$SqlServer uses `[...]`.

Reply:
I now use both quotes again. Thanks for the mssql reminder. I like the way the current state of the code looks and will transform the remaining statements as I find the time to do so.